sim/druid/item_sets_pve.go

```diff
@@ -295,11 +295,13 @@ var ItemSetCenarionRage = core.NewItemSet(core.ItemSet{
 		},
 		// Reduces the cooldown of Enrage by 30 sec and it no longer reduces your armor.
 		4: func(agent core.Agent) {
-			// TODO: Enrage
+			druid := agent.(DruidAgent).GetDruid()
+			druid.Enrage.CD.Duration -= time.Second * 30
 		},
 		// Bear Form and Dire Bear Form increase all threat you generate by an additional 20%, and Cower now removes all your threat against the target but has a 20 sec longer cooldown.
 		6: func(agent core.Agent) {
-			// TODO: Bear, Dire Bear forms
+			druid := agent.(DruidAgent).GetDruid()
+			druid.PseudoStats.ThreatMultiplier += .2
```

Comment:
This is going to apply all the time rather than just in bear form. Maybe we can use an attribute on the druid to track the threat value?